python/requirements-wheel-build.txt

```diff
@@ -0,0 +1,7 @@
+Keras-Preprocessing
+cython>=0.29.11
+numpy==1.14.5; python_version < "3.8"
```

Comment:
In pandas we build against numpy==1.13.3 for python 3.6 (and 1.14.5 for python 3.7), do we have a documented minimum required numpy version for pyarrow?

Reply:
1.14 is pinned in setup.py and originates from https://issues.apache.org/jira/browse/ARROW-3141

I don't think we have a documentation for this.